src/main/java/frc/robot/config/ArmConfig.java

```diff
@@ -94,8 +94,8 @@ public enum ArmPosition {
         GAME_PIECE_BOTTOM
       }
     
-      public static final double TOP_NEO_GEAR_RATIO = Config.robotSpecific(23.5, 0.0, 0.0, 0.0, 0.0, 60.0, 60.0); //comp --> 23.5
-      public static final double BOTTOM_NEO_GEAR_RATIO = 62.5;  
+      public static final double TOP_NEO_GEAR_RATIO = Config.robotSpecific(1.0, 0.0, 0.0, 0.0, 0.0, 60.0, 60.0); // NEO Encoder gear ratio is 23.5;  
```

Comment:
1. Suggest to have a different name to avoid confusion, i.e., don't use NEO_GEAR_RATIO.
2. Why original top and bottom gear ratios are different?